netlink-packet-route/src/rtnl/link/nlas/link_infos.rs

```diff
@@ -478,6 +488,263 @@ impl<'a, T: AsRef<[u8]> + ?Sized> Parseable<NlaBuffer<&'a T>> for InfoKind {
     }
 }
 
+// https://elixir.bootlin.com/linux/v5.9.8/source/drivers/net/bonding/bond_netlink.c#L82
```

Comment:
This website is not official kernel.org and will soon be outdated. There is no need to mention it here.

Reply:
Removed.